pom.xml

```diff
@@ -49,70 +77,78 @@
   <dependencyManagement>
     <dependencies>
       <dependency>
-        <groupId>org.apache.camel.springboot</groupId>
-        <artifactId>camel-spring-boot-dependencies</artifactId>
-        <version>4.0.1</version>
-        <type>pom</type>
-        <scope>import</scope>
+        <groupId>ca.uhn.hapi.fhir</groupId>
+        <artifactId>hapi-fhir-structures-r4</artifactId>
+        <version>${fhir-r4.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.ozonehis</groupId>
+        <artifactId>eip-commons</artifactId>
+        <version>${eip.client.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.projectlombok</groupId>
+        <artifactId>lombok</artifactId>
+        <version>1.18.30</version>
       </dependency>
     </dependencies>
+    <!--        <dependencies>-->
```

Comment:
You can remove this instead of commenting it out unless you intend to put it back